File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/vssdkanimationstatusbar/cs/guids.cs
// Guids.cs
// MUST match guids.h
using System;

namespace Microsoft.VSSDKAnimationStatusBar
{
    static class GuidList
    {
        public const string guidVSSDKAnimationStatusBarPkgString = "24618ee5-46be-42cd-962a-aabb5e61871e";
        public const string guidVSSDKAnimationStatusBarCmdSetString = "2fbd0f3e-37b7-4b2f-a514-e3ddcd8b5c87";

        public static readonly Guid guidVSSDKAnimationStatusBarCmdSet = new Guid(guidVSSDKAnimationStatusBarCmdSetString);
    };
}

File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/vssdkanimationstatusbar/vb/guids.vb
Imports System

Class GuidList
    Private Sub New()
    End Sub

    Public Const guidVSSDKAnimationStatusBarPkgString As String = "2f649bd4-fa93-42f1-9b75-cc312a3ee8cd"
    Public Const guidVSSDKAnimationStatusBarCmdSetString As String = "827528d7-3ceb-4974-92b9-d26848bb3338"

    Public Shared ReadOnly guidVSSDKAnimationStatusBarCmdSet As New Guid(guidVSSDKAnimationStatusBarCmdSetString)
End Class